oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticIndexDefinition.java

```diff
@@ -130,6 +136,7 @@ public ElasticIndexDefinition(NodeState root, NodeState defn, String indexPath,
         this.bulkRetriesBackoff = getOptionalValue(defn, BULK_RETRIES_BACKOFF, BULK_RETRIES_BACKOFF_DEFAULT);
         this.numberOfShards = getOptionalValue(defn, NUMBER_OF_SHARDS, NUMBER_OF_SHARDS_DEFAULT);
         this.numberOfReplicas = getOptionalValue(defn, NUMBER_OF_REPLICAS, NUMBER_OF_REPLICAS_DEFAULT);
+        this.maxResultWindow = getOptionalValue(defn, MAX_RESULT_WINDOW, AEM_QUERY_MAX);
```

Comment:
please do not refer to product names inside the source code

Reply:
Thanks. In parallel, I've also changed the approach to default to the Elasticsearch default value and to keep it configurable.